_includes/events-hero.html

```diff
@@ -0,0 +1,40 @@
+{% macro render(hero) %}
+
+{% from "post-macros.html" import category_icon as category_icon %}
+
+<section class="content_hero hero">
+    <div class="hero_wrapper">
+        <a href="#" class="hero_card hero_card__events">
```

Comment:
This is actually a direct copy of the [existing hero card pattern](https://github.com/cfpb/cfgov-refresh/blob/gh-pages/_includes/hero.html) already in use. The only real change I made was adding the event date.

At first I attempted to add logic for the events hero content in to the existing macro, but due to the differences in how they'll be called (views CPT vs a query for the next event) and the extra logic that will be required to update the events hero as an event becomes live (not yet implemented) I decided to break it into a duplicate macro.

Breaking/updating the existing pattern seemed out of scope for this task, but I agree we could neaten this up a bit.

The event date won't always be a date (the mockups actually show "Watch Now") so I went with the simplest elem (but avoided a `p` because those already have their own styling that would need overriding). We could always add some conditional logic to swap the elem depending on what's to be displayed. Also, `summary_event-date` may not be the best naming choice here. I'm open to suggestions.

I believe `hero_card-padding` includes a margin because there are times the last-child within the elem will have bottom margin, and rather than stacking the two as padding would do, the last-child's margin is ignored (see the hero on `/blog`, there's a `p` with bottom margin).